doc/source/examples/ale_regression_california.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<p>I don't think you need the assumption on the conditional feature distribution for the expression to be true.</p><p> </p><p>learnt -&gt; learned ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Removed the assumption as I think you're right, in any case the sentence didn't clarify much.</p>